src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -93,19 +103,29 @@ const TopicPage = ({ pageData }) => {
               title: curationTitle,
               link,
               position,
+              visualStyle,
             }) => (
-              <Curation
-                headingLevel={curationTitle && 3}
-                key={curationId}
-                visualStyle={VISUAL_STYLE.NONE}
-                visualProminance={visualProminence}
-                promos={summaries}
-                title={curationTitle}
-                topStoriesTitle={topStoriesTitle}
-                position={position}
-                link={link}
-                curationLength={curations && curations.length}
-              />
+              <React.Fragment key={`${curationId}-${position}`}>
+                {shouldRenderMessageBanner(visualStyle, visualProminence) ? (
+                  <MessageBanner
+                    title={curationTitle}
+                    summaries={summaries}
+                    position={position}
+                  />
+                ) : (
+                  <Curation
+                    headingLevel={curationTitle && 3}
+                    visualStyle={visualStyle}
+                    visualProminance={visualProminence}
+                    promos={summaries}
+                    title={curationTitle}
+                    topStoriesTitle={topStoriesTitle}
+                    position={position}
+                    link={link}
+                    curationLength={curations && curations.length}
+                  />
+                )}
```

Comment:
I think this works for now, but would be good to look at refactoring this to make the decision on which component to render a bit clearer. Similar-ish to the way its done in other page types, by having an object of renderable components which are decided upon by certain criteria (visual style + prominence in this case).

Reply:
Thanks for this Aaron, I will add this comment to our Refactoring ticket (**WSTEAMA-220**), and cover it then